client/packages/common/src/intl/locales/fr/common.json

```diff
@@ -2032,6 +2035,7 @@
   "messages.how-to-read-report": "Comment lire ce rapport ?",
   "messages.how-to-read-stock-detail": "Ce rapport contient les mêmes informations que la page de stock, sans aucune interactivité.",
   "messages.how-to-read-stock-status": "Descriptions des statuts :\r\n* Sous-stocké - La MDS est inférieure à la MDS minimale (préférence dépôt : seuil pour sous-stock)\r\n* En excès de stock - La MDS est supérieure à la MDS maximale (préférence dépôt : seuil pour excès de stock)\r\n* En rupture de stock - Le Stock disponible est à 0 et la CMM est supérieur à 0.\r\n* En quantité suffisante - La MDS est entre la MDS minimale et maximale\r\n* Pas de consommation - La CMM est à 0",
+  "messages.how-to-read-inventory-adjustments": "This report displays all inventory adjustments from both finalised stocktakes and direct inventory adjustments. \n\nColumn descriptions: \n* Snapshot Packs: The number of packs recorded at the start of a stocktake (only shown for stocktake adjustments). \n* Counted Packs: The number of packs counted during a stocktake (only shown for stocktake adjustments). \n* Adjustment: The change in stock quantity. For stocktakes this is calculated as Counted - Snapshot. For direct adjustments this is the quantity added (+) or removed (-). \n* Reason: The reason provided for the adjustment.",
```

Comment:
This is still in English, probably best to leave the french translations to the french team, you can just leave this out of the json and they'll pick it up to translate...